app/controllers/BaseController.scala

```diff
@@ -31,6 +31,10 @@ abstract class BaseController(implicit val env: OreEnv,
 
   override val signOns: ModelAccess[SignOn] = this.service.access[SignOn](classOf[SignOn])
 
+  override def notFound()(implicit request: Request[_]) = {
```

Comment:
It's probably possible to delegate this to the ErrorHandler if it was injected but this was easiest. I could change it if you would like